addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -140,6 +138,13 @@ var collision: Node
 var inventory_item: ESCInventoryItem = null setget ,_get_inventory_item
 
 
+# The movable subnode
+var _movable: ESCMovable = null
+
+# The identified animation sprite
+var _animation_sprite: ESCAnimationPlayer = null
```

Comment:
Rename this variable to `_animation_player` to make it clear that it is not an AnimationPlayer nor AnimatedSprite, but a structure that can be any of the two.

Reply:
Done